minutes/2022-03-15.md

```diff
@@ -0,0 +1,32 @@
+# Parsec Community Meeting (15/03/2022) 📋
+
+Minutes of previous meeting are stored
+[here](https://github.com/parallaxsecond/community/tree/main/minutes).
+
+## Attendance
+
+- Anton Antonov
+- Paul Howard
+- Wiktor Kwapisiewicz
+- Ionut Mihalcea
+
+## Agenda
+
+1. Updates on the Parsec 1.0.0 release
+2. Parsec presentation at Open Security Summit
+
+## Minutes
+
+1. Release candidate 3 is expected soon, after feedback from Red Hat
+   - Patrick has opened a PR to remove the dependency on `primal` from the `tss-esapi` crate, as it
+      ballooned the dependency tree; this change will be released as a patch update to 7.0.0
+   - Apart from importing this new version of `tss-esapi`, some of the Parsec service dependencies,
+      particularly the `picky-` crates, have been flagged by Peter Robinson as mismatch with those
+      in `parsec-tool`
+   - The `regex` crate is also being updated to remove [the vulnerability
+      report](https://github.com/parallaxsecond/parsec/issues/587) coming from `cargo-audit`
+   - Once this new RC is out we will begin our testing
+2. Paul held a presentation on Parsec at Open Security Summit - see it
+   [here](https://www.youtube.com/watch?v=JNaMTZI5Fp8)
```

Comment:
We could add it in the `presentations.md` file too :slightly_smiling_face:

Reply:
Was about to do it, but then #79 happened, so I'll leave it to that PR